hedgehog-checkers/src/Hedgehog/Checkers/Properties.hs

```diff
@@ -71,3 +77,28 @@ commutativity f gena = do
 --     classify (x == y) "equals" $
 --     classify (x < y)  "greater than" $
 --     x < y || x == y || x > y
+
+leftDistributive :: ( Ord a
+                    , Eq (f b)
+                    , Show (f a)
+                    , Show (f b)
+                    )
+                 => (forall p q. (p -> q) -> f p -> f q) -> (forall z. f z -> f z -> f z) -> Gen (f a) -> Gen a -> Gen b -> PropertyT IO ()
+leftDistributive x y genFa genA genB = do
+  f <- ordFuncWtf genA genB
```

Comment:
You don't need to change anything, but I really dislike `ordFuncWtf`. It's so slow. if you want more than minimal coverage.

Reply:
I agree, but I don't know what more can be done.
Speaking of that function, I wouldn't mind renaming that module and its contents.

Reply:
>Speaking of that function, I wouldn't mind renaming that module and its contents.

Sure, as long as the new nomenclature retains the useful aposematism.